doc/source/conf.py

```diff
@@ -54,7 +54,7 @@
 
 
 def setup(app):
-    app.add_stylesheet("custom.css")  # may also be an URL
+    app.add_css_file("custom.css")  # may also be an URL
```

Comment:
This is an update for sphinx 4.0. We had a deprecation warning about this for a loooong time.